sdk/communication/azure-communication-rooms/tests/helper.py

```diff
@@ -17,15 +17,4 @@ def process_response(self, response):
         import re
         if 'url' in response:
             response['url'] = re.sub('/identities/[A-Za-z0-9-_%]+', '/identities/sanitized', response['url'])
-        return response
-
-class RequestBodyIdentityReplacer(RecordingProcessor):
```

Comment:
Saw Minnie's comment offline. We don't have to hide them because they are not credentials. Also the id's need to be used as keys of a dictionary so we have to have the actual id's